packages/fx-core/src/plugins/solution/fx-solution/v2/provisionLocal.ts

```diff
@@ -100,6 +107,16 @@ export async function provisionLocalResource(
     return new v2.FxPartialSuccess(localSettings, debugProvisionResult.error);
   }
 
+  if (isConfigUnifyEnabled()) {
+    const localEnvSetupResult = await setupLocalEnvironment(ctx, inputs, envInfo!);
+
+    if (localEnvSetupResult.isErr()) {
+      return new v2.FxPartialSuccess(localSettings, localEnvSetupResult.error);
+    }
+
+    setDataForLocal(envInfo!, localSettings);
```

Comment:
Why copy the local debug config to from `local.state.json` to `localSettings.json` when the feature flag is enable?